docs/shared_parsers_catalog/sysconfig_network.rst

```diff
@@ -0,0 +1,4 @@
+.. automodule:: insights.parsers.sysconfig_network
```

Comment:
Seems it can be put into the `sysconfig.py` file as well, do you have any concern to save it in a single file?

Reply:
Done, moved to the single file